Source/WebKit/PlatformWin.cmake

```diff
@@ -19,6 +19,8 @@ list(APPEND WebKit_SOURCES
     GPUProcess/win/GPUProcessMainWin.cpp
     GPUProcess/win/GPUProcessWin.cpp
 
+    NetworkProcess/NetworkDataTaskDataURL.cpp
```

Comment:
Could you add the same line to Source/WebKit/PlatformPlayStation.cmake as well? They also use CURL.